client/my-sites/checkout/composite-checkout/lib/leave-checkout.ts

```diff
@@ -1,6 +1,8 @@
 import debugFactory from 'debug';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { navigate } from 'calypso/lib/navigate';
+import { addQueryArgs } from 'calypso/lib/route';
+import { QueryArgs } from 'calypso/my-sites/plans/jetpack-plans/types';
```

Comment:
Minor: this can be `import type` since this is not used by any JS code.

```suggestion
import type { QueryArgs } from 'calypso/my-sites/plans/jetpack-plans/types';
```